applications/GeoMechanicsApplication/python_scripts/element_test/test_triaxial/ProjectParameters_stage2.json

```
@@ -0,0 +1,168 @@
+{
+    "problem_data"     : {
+        "problem_name"  : "triaxial",
+        "parallel_type" : "OpenMP",
+        "echo_level"    : 1,
+        "start_time"    : 0.0,
+        "end_time"      : 1.0
+    },
+    "solver_settings"  : {
+        "time_stepping"                      : {
+            "time_step"             : 0.05,
+            "max_delta_time_factor" : 1
+        },
+        "solver_type"                        : "U_Pw",
+        "solution_type"                      : "Quasi-Static",
+        "strategy_type"                      : "line_search",
+        "scheme_type"                        : "Backward_Euler",
+        "model_part_name"                    : "PorousDomain",
+        "domain_size"                        : 2,
+        "echo_level"                         : 1,
+        "model_import_settings"              : {
+            "input_type"     : "mdpa",
+            "input_filename" : "triaxial"
```

Since the mesh doesn't change we should be able to use the following setting:
```suggestion
            "input_type"     : "use_input_model_part"
```

---

I will be converting this test to a single stage because of the imposed stress state that is being applied now.